README.md

```diff
@@ -58,6 +58,44 @@ spec:
             - Resource
 ```
 
+
```

Comment:
Add this under a new heading called `### Artifact Hub`

Reply:
Add a new bullet point where L30 currently is that states:

"* Create the `artifacthub-pkg.yml` file in the same directory as your policy. See the Artifact Hub section below for more details on its contents."

Reply:
Find the appropriate bullet above in the same Contribution heading to state the following things:

1. A dedicated folder must be created for each policy.
2. The folder must be named the same as the policy.
3. Use dashes and not underscores.